File: username.py
#Takes username as Input
#Needs the following dependencies :
# pip install requests[security]
# pip install termcolor

import requests
from termcolor import colored

username = raw_input("Enter the username : ")
sites = "facebook twitter instagram github youtube soundcloud tumblr".split()


def check(status_code):
	if status_code == 404:
		print(colored('Available', 'green'))
	else: 
		print(colored('Taken', 'red'))


for site in sites:
	if site == 'tumblr':
		url = "https://{}.{}.com/".format(username, site)
	else:	
		url = "https://{}.com/{}".format(site, username)
	print(url)
	check(requests.get(url).status_code)

File: username.js
var sites = ["facebook", "twitter", "instagram", "github", "youtube", "soundcloud"];

// request.onreadystatechange = function(){

// 	};

var request = new XMLHttpRequest();
var username = prompt("enter your username ");
var i = 0;
for( ; i < sites.length ; i++) {
	console.log(i);
	//function invocation
	//var yolo = "https://".concat(sites[i]).concat(".com/").concat("gyanl")
	var yolo = "https://"+sites[i]+".com/"+username;
	console.log(yolo);
	xhr = new XMLHttpRequest();
	xhr.open("HEAD", "http://instagram.com/gyanl" ,true);
	xhr.onreadystatechange=function() {
		alert("HTTP Status Code:"+xhr.status)
	}
	xhr.send(null);  
	

	

}






